### DISCUSSION TITLE
Text should have max width of size-30

### DISCUSSION THREAD

**Discussion by pradyuk** [twilio-labs/paste#1103] [General] [answered]
I am trying to get the account and subaccount title in the masthead to truncate with ellipses and tooltip on hover. I am using the tooltip prop which works well, but not sure how to address the "truncate with ellipse". Any examples will be helpful.

**Comment by corinne-nagel** [comment]
Hi @pradyuk could you please provide some screenshots/code of what you're talking about so we can give accurate guidance?

**Reply by pradyuk** [reply]
Sure, pasting the screenshot here. 
![Uploading Screen Shot 2021-01-20 at 4.24.16 PM.png…]()

**Comment by richbachman** [comment, marked as answer]
Hi @pradyuk Paste has a Truncate component that might be helpful here. Take a look at the examples and see if you can find the information there: https://paste.twilio.design/components/truncate#basic-truncate

**Reply by pradyuk** [reply]
Thanks @richbachman. That worked, but using truncate is changing the alignment of the text from center (whats seen in yellow).
![Screen Shot 2021-01-20 at 4 57 49 PM](https://user-images.githubusercontent.com/11803735/105259472-b7865200-5b40-11eb-93a2-0fd307e1083e.png)

Code looks like:
{parentAccount ? (
                <Box
                  paddingLeft="space40"
                  paddingRight="space40"
                  marginLeft="space30"
                  fontSize="fontSize20"
                  lineHeight="lineHeight20"
                  backgroundColor="colorBackgroundSubaccount"
                  style={{ borderRadius: "12px" }}
                  data-testid={accountDropdownName.sid}
                >
                  <Tooltip
                    text={accountDropdownName.friendlyName}
                    placement="bottom"
                  >
                    <Box maxWidth="size30">
                      <Text
                        as="span"
                        fontSize="fontSize20"
                        lineHeight="lineHeight20"
                        fontWeight="fontWeightBold"
                      >
                        <Truncate>{accountDropdownName.friendlyName}</Truncate>
                      </Text>
                    </Box>
                  </Tooltip>
                </Box>
              ) : null}

**Reply by richbachman** [reply]
@pradyuk can I see a more complete code sample? If not, you might try to change the display prop on Box. Maybe try `inline-block` or `inline-flex`. I'm not sure what exactly would help without seeing more of a code sample. You can also drop the `style={{ borderRadius: "12px" }}` and use one of our `BorderRadius` tokens: https://paste.twilio.design/tokens/#radii

**Reply by pradyuk** [reply]
Thanks @richbachman! inline-flex worked! :)
